src/ManifestStrategy/AbstractManifestStrategy.php

```diff
@@ -74,4 +77,60 @@ protected function askValue(string $question, array $choices, InputOutputInterfa
 
         return $value;
     }
+
+    /**
+     * @param array $manifest
+     *
+     * @return \SprykerSdk\Integrator\Transfer\ClassMetadataTransfer
+     */
+    protected function createClassMetadataTransfer(array $manifest): ClassMetadataTransfer
+    {
+        $transfer = (new ClassMetadataTransfer())
```

Comment:
What do you think about of creation of some kind of plugin stack expander and make the current functionality extendable?

Reply:
From my perspective of view, we don't need a plugin stack when we have only two properties in the transfer. 
Maybe when there will be 10)

Reply:
`ClassMetadataTransfer` contains 10 properties and we will extend it in the future.
That's why it will be useful to provide an easy and convenient extension path.